hub-detect/src/main/groovy/com/blackducksoftware/integration/hub/detect/workflow/PhoneHomeManager.java

```diff
@@ -84,22 +86,38 @@ public void startPhoneHome() {
     }
 
     public void startPhoneHome(final Set<BomToolGroupType> applicableBomToolTypes) {
-        performPhoneHome(applicableBomToolTypes);
+        final Map<String, String> metadata = new HashMap<>();
+        if (applicableBomToolTypes != null) {
+            final String applicableBomToolsString = applicableBomToolTypes.stream()
+                    .map(it -> it.toString())
+                    .collect(Collectors.joining(","));
+            metadata.put("bomToolTypes", applicableBomToolsString);
+        }
+        performPhoneHome(metadata);
     }
 
-    private void performPhoneHome(final Set<BomToolGroupType> applicableBomToolTypes) {
+    public void startPhoneHome(final Map<BomToolGroupType, Long> applicableBomToolTimes) {
+        final Map<String, String> metadata = new HashMap<>();
+        if (applicableBomToolTimes != null) {
+            final String applicableBomToolsString = applicableBomToolTimes.keySet().stream()
+                    .map(it -> it.toString() + ":" + applicableBomToolTimes.get(it))
```

Comment:
Would string formatting be more appropriate here?

Reply:
Done